tests/banks.py

```diff
@@ -100,6 +100,60 @@ def test_success_fetch_validators(requests_mock):
     assert response == result
 
 
+def test_success_fetch_banks(requests_mock):
+    result = {
+        'count': 3,
```

Comment:
shouldn't the count be 1 as well is the record is showing only 1?

Reply:
Yeah